src/blosc2/schunk.py

```diff
@@ -463,17 +478,16 @@ def decompress_chunk(self, nchunk, dst=None):
         Examples
         --------
         >>> import blosc2
-        >>> cparams = {'typesize': 1}
-        >>> storage = {'cparams': cparams}
-        >>> schunk = blosc2.SChunk(chunksize=11, **storage)
+        >>> schunk = blosc2.SChunk(cparams = {'typesize': 1})
         >>> buffer = b"wermqeoir23"
-        >>> schunk.append_data(buffer)
+        >>> print(schunk.append_data(buffer))
         1
-        >>> schunk.decompress_chunk(0)
+        >>> print(schunk.decompress_chunk(0))
         b'wermqeoir23'
+        >>> # Construct a mutable bytearray object
         >>> bytes_obj = bytearray(len(buffer))
         >>> schunk.decompress_chunk(0, dst=bytes_obj)
-        >>> bytes_obj == buffer
+        >>> print(bytes_obj == buffer)
```

Comment:
No need to use print.  Please try to avoid as many `print` statements as possible.  With modern f-strings, prints in these examples are almost unnecessary.